{"name": "用户手机号登录", "status": "failed", "statusDetails": {"message": "AssertionError: assert False is True\n +  where False = <class 'core.ResultBase.ResultResponse'>.success", "trace": "self = <testcases.usercenter.test_user.TestUser object at 0x10b5d2be0>\nusername = 17380900323, password = 123456\n\n    @pytest.mark.parametrize(\"username,password\", get_data['user_login'])\n    @allure.story(\"用户登录\")\n    @allure.title(\"用户手机号登录\")\n    def test_login(self, username, password):\n        print(username, password)\n        result = login(username, password)\n>       assert result.success is True\nE       AssertionError: assert False is True\nE        +  where False = <class 'core.ResultBase.ResultResponse'>.success\n\ntest_user.py:39: AssertionError"}, "attachments": [{"name": "log", "source": "8f7490f1-8398-477d-8c50-215e68763d0b-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "868ad92d-f09b-46aa-a569-3434b46e451b-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "0818a523-1741-406f-9a7e-5fd4409c93a7-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "17380900323"}, {"name": "password", "value": "123456"}], "start": 1659076899752, "stop": 1659076899862, "uuid": "4cb7c064-3735-4693-b0d6-69a4ae31f75c", "historyId": "f70372c1e8a5bb8d0ba966d1855d2aad", "testCaseId": "18cc3eb427b1ac4ba4cd5ce6c4a3501c", "fullName": "testcases.usercenter.test_user.TestUser#test_login", "labels": [{"name": "story", "value": "用户登录"}, {"name": "feature", "value": "用户中心模块"}, {"name": "parentSuite", "value": "testcases.usercenter"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "shallowdeMacBook-Air.local"}, {"name": "thread", "value": "50509-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.usercenter.test_user"}]}